packages/backend/src/graphql/task/task.ts

```diff
@@ -44,13 +44,25 @@ export const Task = builder.prismaObject('Task', {
       args: {
         from: t.arg({ type: DateScalar, required: true }),
         to: t.arg({ type: DateScalar, required: false }),
+        forUserId: t.arg.id({
+          required: false,
+          description:
+            'Filter work hours where the given user is a project member. If not given, the work hours of the signed in user are returned',
+        }),
       },
-      resolve: async (task, _arguments, context) => {
-        const interval = { start: _arguments.from, end: _arguments.to ?? _arguments.from }
+      // when signed in user requests work hours for another user, the signed in user must be an admin
+
+      authScopes: (task, { forUserId }, context) => {
+        const showWorkHoursForOtherUser = !!forUserId && forUserId !== context.session.user.id
```

Comment:
```suggestion
        const showWorkHoursForOtherUser = forUserId && forUserId !== context.session.user.id
```
&& should handle the truthiness check, so there is no need in double negation

Reply:
I needed to explicitly convert it to a boolean with double negation so that it is either true or false.
This would then much when reused in the `getWhereUserIsMember().`